game.html

```diff
@@ -185,6 +231,13 @@
             <!-- animation="property: object3D.position.y; dir: alternate; from: 5.55; to: 5.7 ; dur: 2000; easing: easeInOutQuad; loop: true"  -->
 
             <!-- player -->
+            <a-box id="player-collision"
+                dynamic-body="shape: sphere; sphereRadius: 0.65"
+                material="opacity: 0"
+                position="14 5.55 0"
+                arrows-controls
+                adjust-player-position
```

Comment:
player-collision sphere should be restored on collision not every frame

Reply:
It needs to be set every frame for it to correctly detect collision AND take into account player movement.

Reply:
Although it doesn't need to be done twice every frame :facepalm: